packages/react-components/src/atoms/Link.tsx

```diff
@@ -64,6 +64,7 @@ interface NormalLinkProps {
   readonly small?: undefined;
   readonly enabled?: undefined;
   readonly noMargin?: undefined;
+  readonly tabletFullWidth?: undefined;
```

Comment:
I don't like it being so specific

Reply:
agreed. It was something I wanted to discuss.

Agreed, changed now to fullWidth